app/src/main/java/fr/free/nrw/commons/media/ZoomableActivity.kt

```diff
@@ -650,4 +659,13 @@ class ZoomableActivity : BaseActivity() {
         scope.cancel()
         super.onDestroy()
     }
+
+    companion object {
+        /**
+         * Key for Accessing Intent Data Named "Origin", The value indicates what fragment
+         * ZoomableActivity was created by. It is null if ZoomableActivity was created by
+         * the custom picker.
+         */
+        private const val ORIGIN_LABEL = "Origin";
```

Comment:
Sorry I missed this. Any reason for calling this "label"?
How about just `ORIGIN`?

Reply:
Done